keyboards/keychron/c3_pro/config.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2023 @ Keychron(https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Indication led */
+#define LED_MAC_OS_PIN A4
+#define LED_WIN_OS_PIN A5
+#define LED_OS_PIN_ON_STATE 1
+
+/* EEPROM Driver Configuration */
+#define WEAR_LEVELING_LOGICAL_SIZE 2048
+#define WEAR_LEVELING_BACKING_SIZE (WEAR_LEVELING_LOGICAL_SIZE * 2)
```

Comment:
This can be moved into `info.json` as:
```json
"eeprom": {
        "wear_leveling": {
            "backing_size": 4096
        }
    },
```
`logical_size` doesn't need to be defined since `backing_size` will be double that amount.